services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -196,6 +197,9 @@ class HankeEntity(
     )
     var tormaystarkasteluTulokset: MutableList<TormaystarkasteluTulosEntity> = mutableListOf()
 
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "hanke", cascade = [])
```

Comment:
What does cascade = [] mean? No cascade is a default and it does not need to be defined at all (I think).

Reply:
Yes it is the default. Redundancy removed.

Reply:
I do not see any changes in this.